src/frontend/src/handler/extended_handle.rs

```diff
@@ -18,19 +18,33 @@ use bytes::Bytes;
 use pgwire::types::Format;
 use risingwave_common::error::{ErrorCode, Result};
 use risingwave_common::types::DataType;
-use risingwave_sqlparser::ast::Statement;
+use risingwave_sqlparser::ast::{Query, Statement};
 
-use super::{query, HandlerArgs, RwPgResponse};
+use super::{handle, query, HandlerArgs, RwPgResponse};
 use crate::binder::BoundStatement;
 use crate::session::SessionImpl;
 
-pub struct PrepareStatement {
+#[derive(Clone)]
+pub enum PrepareStatement {
+    Prepared(PreparedResult),
+    PureStatement(Statement),
+}
```

Comment:
Could you please clarify in which circumstances we will use `PureStatement` and add some comments? IIUC, it seems to be used by `Create` statements?

Reply:
I refine them in another PR.